build_aws.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Let's just put these dependency build steps in the `README` like [pipeline_kafka](https://github.com/pipelinedb/pipeline_kafka) does for `librdkafka`.